src/devices/SoftPwm/SoftwarePwmChannel.cs

```diff
@@ -15,22 +15,20 @@ namespace System.Device.Pwm.Drivers
     public class SoftwarePwmChannel : PwmChannel
     {
         private readonly bool _shouldDispose;
-        // use to determine the freqncy of the PWM
-        // PulseFrequency = total frenquency
-        // curent pulse width = when the signal is hi
-        private double _currentPulseWidth;
-        private double _pulseFrequency;
+        // how long the signal is high in its period
+        private double _pulseWidthMs;
+        private double _periodMs;
         private int _frequency;
         // Use to determine the length of the pulse
-        // 100 % = full output. 0%= nothing as output
-        private double _percentage;
+        // 100% = 1.0 = full output. 0% = 0.0 = nothing as output
+        private double _dutyCycle;
```

Comment:
I am not sure if this is a better name than the percentage.

Reply:
we call argument and property DutyCycle (+/- casing) so this is just consistency